src/server/routes.js

```diff
@@ -99,6 +99,28 @@ var Routes = function (app) {
     }
   });
 
+  /**
+  * POST /locations
+  */
+  app.post('/locations/:company_token', async function (req, res) {
+    var auth = req.get('Authorization');
+
+    req.body.params = req.body.params || {};
+    req.body.params.company_token = req.params.company_token;
+
+    console.log('POST /locations/:company_token\n%s'.green, JSON.stringify(req.headers, null, 2));
```

Comment:
Interpolate the value of `req.params.company_token` in the logged url.

```javascript
console.log('POST /locations/%s\n%s'.green, req.params.companh_JSON.stringify(req.headers, null, 2));
```

Reply:
ok!